dvc/api/__init__.py

```diff
@@ -29,7 +29,7 @@ def get_url(path, repo=None, rev=None, remote=None):
             raise OutputNotFoundError(path, repo)
 
         cloud = metadata.repo.cloud
-        hash_info = get_hash(path_info, _repo.repo_fs, "md5")
+        hash_info = get_hash(path_info, _repo.repo_fs, "md5", _repo.odb)
```

Comment:
This temporary and will be `fs.info(path_info).get("md5")` in the future, so nevermind the ugliness.